core/src/it/scala/org/ensime/indexer/SearchServiceSpec.scala

```diff
@@ -225,7 +362,7 @@ object SearchServiceTestUtils {
   def refresh()(implicit service: SearchService): (Int, Int) =
     Await.result(service.refresh(), Duration.Inf)
 
-  def searchClasses(expect: String, query: String)(implicit service: SearchService) = {
+  def searchClasses(expect: String, query: String)(implicit service: SearchService, p: Position) = {
```

Comment:
interesting, I thought the implicit `Position`s were in the 2.0 branch. I must have missed this test.